openapi/components/schemas/user-readableFields.yaml

```diff
@@ -17,16 +17,18 @@ properties:
   groups:
```

Comment:
Groups should look something like:
```
  groups:
    description: |
      This user's group memberships.
    type: array
    items:
      type: object
      title: Group Membership
      description: |
          A group within Schoolbox.
          
          This contains some of the folder's fields in this context.
      properties:
          $ref: ../schemas/userGroupMembership-readableFields.yaml
``` 
Don't see why `additionalProperties` is needed here alas, this is just after a quick look out of context.

Docs will look like:
<img width="800" alt="image" src="https://user-images.githubusercontent.com/37988348/232959106-b62ea0ab-2f9f-4d7c-b354-a9a3e6f947ee.png">

Reply:
Thanks @Brahmah, I agree.  Fixed.